sites/bizbash.com/server/templates/index.marko

```diff
@@ -29,143 +29,102 @@ $ const { id, alias, name, pageNode } = data;
     </marko-web-resolve-page>
   </@above-container>
   <@page>
-        <marko-web-query|{ nodes }|
-          name="website-optioned-content"
-          params={ sectionId: id, optionName: "Pinned", limit: 5, requiresImage: true, queryFragment }
+    <marko-web-query|{ nodes }|
+      name="website-optioned-content"
+      params={ sectionId: id, optionName: "Pinned", limit: 5, requiresImage: true, queryFragment }
+    >
+      <global-content-hero-flow nodes=nodes />
+    </marko-web-query>
+    <marko-web-gam-display-ad id="gpt-ad-lb1" modifiers=["above-container"] />
+    <div class="col-lg-12 mb-block">
+      <marko-web-query|{ nodes }|
+        name="website-scheduled-content"
+        params={ sectionAlias: "industry-buzz", limit: 3, queryFragment }
+      >
+        <div class="node-list__header">
+            <marko-web-link href="/industry-buzz">Industry Buzz</marko-web-link>
+        </div>
+        <global-content-card-deck-flow nodes=nodes />
+      </marko-web-query>
+    </div>
+    <div class="col-lg-12 mb-block">
+      <marko-web-query|{ nodes }|
+        name="website-scheduled-content"
+        params={ sectionAlias: "production-strategy", limit: 3, queryFragment }
+      >
```

Comment:
For the rest of these block/flows, using the new card display, I would add the `requiresImage: true` parameter to the marko-web-query block.  This will insure that all three cards display as they would expect.

Reply:
Apologies for not getting to this earlier only just now saw the changes request